linode_api4/objects/linode.py

```diff
@@ -870,6 +883,32 @@ def transfer(self):
 
         return self._transfer
 
+    @property
+    def placement_group(self) -> Optional[PlacementGroup]:
```

Comment:
Surely there's a way better way to do this, but for the time being we don't have a way to transform an arbitrary ID into an object like we can with derived classes & slugs.